packages/server-ct/example/src/components/Prettiest.spec.js

```diff
@@ -2,30 +2,31 @@
 
 // import '../main'
 import HelloWorld from './HelloWorld'
-import { mount } from '@vue/test-utils'
+import { mount } from '@cypress/vue'
 import Vue from 'vue'
-// import Vue from 'vue'
 
 Vue.config.productionTip = false
 
 describe('Prettiest', () => {
-  it('spec works', async () => {
-    // window.Vue = Vue
-    // new Vue(HelloWorld).$mount('#root')
-    const wrapper = mount(HelloWorld, { attachTo: '#__cy_app' })
+  it('spec works', () => {
+    mount(HelloWorld, {
+      attachTo: '#__cy_app',
```

Comment:
If we are using mount from cypress vue, we "should" not need this esoteric mounting point, should we?

Reply:
Actually we just need to update @cypress/vue not to blow away the AUT and instead mount components inside of `#__cy_app`

Reply:
somewhat surprisingly I removed the `attachTo` and everything seems to work just fine - seems like this was accounted for somewhere else. 🎉